torch/distributed/algorithms/ddp_comm_hooks/debugging_hooks.py

```diff
@@ -8,8 +8,7 @@
 
 def noop_hook(_: Any, bucket: GradBucket) -> torch.futures.Future[torch.Tensor]:
     """
-    This DDP communication hook returns a future that wraps the input,
-    so it is a noop that does not incur any communication overheads.
+    Return a future that wraps the input, so it is a noop that does not incur any communication overheads.
```

Comment:
nit (before this PR):
```suggestion
    Return a future that wraps the input, so it is a no-op that does not incur any communication overheads.
```